packages/utils/sidebar.ts

```diff
@@ -19,6 +23,33 @@ import walletSVG from "../../assets/icons/wallet-sidebar.svg";
 import { SidebarRecordType } from "../components/navigation/types";
 import { NetworkKind } from "../networks";
 
+// TODO: Maybe handle lightTheme directly in a future icons system
+export const RWA_SIDEBAR_LIST: (isLightTheme: boolean) => SidebarRecordType = (
+  isLightTheme,
+) => {
+  return {
+    "rwa-home": {
+      title: "RWA Home",
+      route: "RWAHome",
+      id: "RWAHome",
+      icon: isLightTheme ? lightLaunchpadSVG : launchpadSVG,
+    },
+
+    "rwa-portfolio": {
+      title: "RWA Portfolio",
+      route: "RWAHome",
+      id: "RWAPortfolio",
+      icon: isLightTheme ? lightMarketplaceSVG : marketplaceSVG,
+    },
+    "rwa-wallet": {
+      title: "RWA Wallet",
+      route: "RWAHome",
```

Comment:
TODO: Add RWAWallet into `Navigator`
```suggestion
      route: "RWAWallet",
```